scripts/install_third_party.py

```diff
@@ -341,6 +341,48 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def install_elasticsearch_dev_server():
+    """This installs a local ElasticSearch server to the oppia_tools
+    directory to be used by development servers and backend tests.
+    """
+    try:
+        subprocess.call(
+            ['%s/bin/elasticsearch' % common.ES_PATH, '--version'],
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+        python_utils.PRINT('ElasticSearch is already installed.')
+        return
+    except OSError:
+        python_utils.PRINT('Installing ElasticSearch...')
+
+    if common.is_mac_os() or common.is_linux_os():
+        file_ext = 'tar.gz'
+        def download_and_extract(*args):
+            """This downloads and extracts the elasticsearch files."""
+            download_and_untar_files(*args)
```

Comment:
can we do download_and_extract_function = download_and_untar_files? (not sure!)

Reply:
It works in the code, but it doesn't work in the unit tests which check that the right extract function is called for each OS option, since `download_and_untar_files` is called by its alias instead of directly. I tried changing the test to check if `download_and_extract_function` was called, but I feel like that defeats the purpose of the test.